.coveragerc

```diff
@@ -23,6 +23,7 @@ exclude_lines =
   if _t.TYPE_CHECKING:
   if t.TYPE_CHECKING:
   @overload
+  class .*\bProtocol\b.*\):
```

Comment:
Huh, why add this (I really really don't want to read through all the conversation on this PR and the blame doesn't help :sob:)

Reply:
Nevermind I figured it out.

Reply:
Yeah otherwise coverage complains that the method stubs weren’t run. Though maybe we should rethink this, if we add default implementations to any protocols in the future it won’t flag those.